vision/fiducial_detection_and_pose.cc

```diff
@@ -9,11 +9,17 @@ namespace {
 MarkerRvecsTvecs rvecs_tvecs_from_corners(
     const std::vector<std::vector<cv::Point2f>>& corners) {
   std::vector<cv::Vec3d> rvecs, tvecs;
+
   const cv::Mat camera_matrix =
-      (cv::Mat1d(3, 3) << 320, 0, 320, 0, 320, 320, 0, 0, 1);
-  const cv::Mat distortion_coefficients =
-      (cv::Mat1d(1, 8) << 0, 0, 0, 0, 0, 0, 0, 0);
-  cv::aruco::estimatePoseSingleMarkers(corners, 0.49375, camera_matrix,
+      (cv::Mat1d(3, 3) << 533.086677901258, 0, 318.6124474231535,
+                          0, 526.7508307011435, 233.3218572898686,
+                          0, 0, 1);
+  const cv::Mat distortion_coefficients = (cv::Mat1d(1, 5) << 0.1957195456695698,
+                                                              -0.738918431892389,
+                                                              0.004904687104832405,
+                                                              0.01328395230082144,
+                                                              0.7438835398485716);
+  cv::aruco::estimatePoseSingleMarkers(corners, 0.1335, camera_matrix,
```

Comment:
could you pull out `0.1335` as `constexpr double SIZE_M = ...`

We should at some point add runtime configuration for this, but for now just to make it more convenient for us to operate on differently sized fiducials